packages/cli/oclif.manifest.json

```diff
@@ -209,15 +209,15 @@
         "no-json-output": {
```

Comment:
I closed my other ticket https://github.com/Shopify/hydrogen/pull/1644

`public-deployment` flag will need to be removed before releasing `deploy`.